site/content/docs/4.3/examples/cover/index.html

```diff
@@ -3,33 +3,32 @@
 title: Cover Template
 extra_css:
   - "cover.css"
-body_class: "text-center"
+html_class: "h-100"
+body_class: "text-center text-white bg-dark d-flex h-100"
 include_js: false
 ---
 
 <div class="cover-container d-flex w-100 h-100 p-3 mx-auto flex-column">
   <header class="masthead mb-auto">
-    <div class="inner">
-      <h3 class="masthead-brand">Cover</h3>
-      <nav class="nav nav-masthead justify-content-center">
+    <div>
+      <h3 class="masthead-brand mb-0 float-md-left">Cover</h3>
```

Comment:
```suggestion
      <h3 class="float-md-left mb-0">Cover</h3>
```

Since `.masthead-brand` is unused, we can drop it.